subsys/bluetooth/host/hci_core.c

```diff
@@ -1849,6 +1849,33 @@ static void le_read_all_remote_feat_complete(struct net_buf *buf)
 }
 #endif /* CONFIG_BT_LE_EXTENDED_FEAT_SET */
 
+#if defined(CONFIG_BT_FRAME_SPACE_UPDATE)
+static void le_frame_space_update_complete(struct net_buf *buf)
+{
+	struct bt_hci_evt_le_frame_space_update_complete *evt = (void *)buf->data;
+	struct bt_conn *conn;
+	struct bt_conn_le_frame_space_updated params;
+	uint16_t handle = sys_le16_to_cpu(evt->handle);
+
+	conn = bt_conn_lookup_handle(handle, BT_CONN_TYPE_LE);
+	if (conn == NULL) {
+		LOG_ERR("Unknown conn handle 0x%04X", handle);
+		return;
+	}
+
+	params.status = evt->status;
+
+	if (params.status == BT_HCI_ERR_SUCCESS) {
```

Comment:
The docs in the header says:
> The remaining parameters will be unchanged if status is not @ref BT_HCI_ERR_SUCCESS.

But the fields are not set so they will contain garbage. Should the api description be updated to say the rest of the fields are invalid in case of an HCI error?

Alternatively we would need to store the current frame space values somewhere in host and populate `params` with those, but that seems inefficient.